### ISSUE TITLE
Not listed option on whitelist

### ISSUE THREAD

**Issue by andresmijares** [benkeen/react-country-region-selector#7]
Any chance to create some kind of whitelist plus an Not listed, value?

**Comment by benkeen** [comment]
Hey @andresmijares  - I don't quite follow. There's both a `blacklist` and `whitelist` option to let you pinpoint what countries get included. Does one of those work for you?

**Comment by andresmijares** [comment]
yeah! both work perfect but, I have an use case that goes as follows:

US
CAN
ARG
Not Listed

I need to catch that Not listed as well, does it support this case? :) 

Ty for ur answer!
